src/utils/venue.ts

```diff
@@ -29,6 +34,9 @@ export const buildEmptySpace = (
   const list = new DataTransfer();
 
   const fileList = list.files;
+  const isSpaceWithTables =
+    template === VenueTemplate.jazzbar ||
+    template === VenueTemplate.conversationspace;
```

Comment:
This can be improved later on with addition of a capabilities map, but for now it's good enough.